.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
please don't commit your editor config

Reply:
should we add this to .gitigniore?

Reply:
I considered it. But our modulesync_config already purges it: https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L165-L166. So you could add it to the .gitignore, but it will be deleted when you run msync, and I think that would cause more confusion.